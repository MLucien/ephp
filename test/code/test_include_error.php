<?php

include(__DIR__ . "/function_broken.php");
include(__DIR__ . "/class_broken.php");

broken();
$broken = new broken;
$broken->broken();
