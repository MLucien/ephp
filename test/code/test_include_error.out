
Warning: strpos() expects parameter 1 to be string, array given in {{CWD}}/test/code/function_broken.php on line 4

Warning: strpos() expects parameter 1 to be string, array given in {{CWD}}/test/code/class_broken.php on line 5

Warning: strpos() expects parameter 1 to be string, array given in {{CWD}}/test/code/class_broken.php on line 5
